sdk-base/src/main/java/com/mapbox/maps/plugin/annotation/GeoJsonOptions.kt

```diff
@@ -0,0 +1,68 @@
+package com.mapbox.maps.plugin.annotation
+
+/**
+ * Configure class for composing GeoJsonSource objects that included in Annotation
+ * @see <a href="https://docs.mapbox.com/mapbox-gl-js/style-spec/sources/#geojson">The online documentation</a>
+ */
+data class GeoJsonOptions(
```

Comment:
Adding this class to SDK-base and making it not fully compatible with style extension will be confusing for users that are migrating from the old API. Since it's only used for annotations, what about we rename this class to `AnnotationSourceOptions` and move it to annotation plugin itself?

Reply:
We will need to keep this class in sdk-base as it is part of the API, move it to annotation plugin will make it not available for users.